web/themes/custom/pineapple/pineapple.theme

```diff
@@ -55,6 +55,14 @@ function pineapple_preprocess_node(array &$variables) {
     $owner = $node->getOwner();
     $variables['owner'] = $owner;
   }
+
+  if ($node->getType() == 'article' || 'recipe') {
```

Comment:
I’m pretty sure this code won’t run for recipes, you’ll need to add the node->get type after the or

Reply:
done